src/dispatch/incident/flows.py

```diff
@@ -469,6 +498,19 @@ def conversation_topic_dispatcher(
             individual_id=individual.id,
         )
 
+    description, details = check_for_tag_change(
+        previous_incident_tags=previous_incident.tags, current_incident_tags=incident.tags
+    )
+    if description:
+        event_service.log_incident_event(
+            db_session=db_session,
+            source="Incident Participant",
+            description=f"{individual.name} {description}",
+            details=details,
+            incident_id=incident.id,
+            individual_id=individual.id,
+        )
```

Comment:
Should this logic live in the `incident_update_flow` function? This function is dedicated to determine if the conversation topic needs to be updated.